### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EasyRhinoMSFT**
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
+# Microsoft SARIF Website
 
-# Contributing
```
Comment (eddynaka) [inline README.md:2]: >Contributing [](http://example.com/codeflow?start=2&length=12)

I think we should move this content to the CONTRIBUTING.md file.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+- Web-based utility for viewing and validating SARIF log files
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+## [Microsoft SARIF website](https://sarifweb.azurewebsites.net)
```
Comment (eddynaka) [inline README.md:20]: >Microsoft SARIF website [](http://example.com/codeflow?start=4&length=23)

Remove this and move the link to the first line

File: README.md [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+## Find Links to Resources
+- SARIF's OASIS open standard
+  - Specification
+  - Tutorials
+  - GitHub project
+- Tools and libraries for developers
+  - Microsoft's open source SARIF SDK for .NET applications
+  - NuGet package for third-party Visual Studio extension interop
+  - Python module
```
Comment (eddynaka) [inline README.md:14]: >Python module [](http://example.com/codeflow?start=4&length=13)

should we also add sarif-js-sdk?

Comment (EasyRhinoMSFT) [reply README.md:14]: Sure but I don't see that linked on the site. Where does it live?

File: README.md (hunk 2 above)
Comment (EasyRhinoMSFT) [reply README.md:20]: I think you mean just move this to the top and keep the "Microsoft SARIF website" text

File: README.md (hunk 3 above)
Comment (EasyRhinoMSFT) [reply README.md:14]: NM found it. I'll add this to the site

**Review by eddynaka — APPROVED** [review]
:shipit:

**Pull request merged**